doc/Android.md

```diff
@@ -0,0 +1,109 @@
+**Document based on Minetest `5.6.1` Android build**
+# Minetest: Android version
+Minetest for Android are the same Minetest on Windows but compiled for Android, not seperate version
```

Comment:
You mention the "Android version" and yet "no separate version".
I'd just remove this sentence. If there were another repository, it would be necessary to link it appropriately pretty much everywhere.